### ISSUE TITLE
Exception when using assembly-informational-version in GitVersion.yml

### ISSUE THREAD

**Issue by brunobertechini** [GitTools/GitVersion#1048]
Iam having the exception below when trying to override assembly-informational-version in GitVersion.yml.

Exception happens either when using GitVersion.exe and also when building using VSTS Team Build Server.

This is the content of GitVersion.yml
`assembly-informational-version: '{NugetVersion}'`

Exception:

```
INFO [09/27/16 18:29:37:45] Working directory: C:\My\Path\To\Repository
INFO [09/27/16 18:29:37:46] IsDynamicGitRepository: False
INFO [09/27/16 18:29:37:49] Returning Project Root from DotGitDirectory: C:\My\Path\To\Repository\.git - C:\My\Path\To\Repository
INFO [09/27/16 18:29:37:49] Running on Windows.
INFO [09/27/16 18:29:37:49] IsDynamicGitRepository: False
INFO [09/27/16 18:29:37:49] Returning Project Root from DotGitDirectory: C:\My\Path\To\Repository\.git - C:\My\Path\To\Repository
INFO [09/27/16 18:29:37:49] Project root is: C:\My\Path\To\Repository
INFO [09/27/16 18:29:37:49] DotGit directory is: C:\My\Path\To\Repository\.git
INFO [09/27/16 18:29:37:50] IsDynamicGitRepository: False
INFO [09/27/16 18:29:37:50] Returning Project Root from DotGitDirectory: C:\My\Path\To\Repository\.git - C:\My\Path\To\Repository
INFO [09/27/16 18:29:37:56] Begin: Loading version variables from disk cache
  INFO [09/27/16 18:29:37:57] Cache file C:\My\Path\To\Repository\.git\gitversion_cache\3F67CE2916EE2FC1D954442039D9D3781D32BCD9.yml not found.
INFO [09/27/16 18:29:37:57] End: Loading version variables from disk cache (Took: 2.00ms)
INFO [09/27/16 18:29:37:57] IsDynamicGitRepository: False
INFO [09/27/16 18:29:37:57] Returning Project Root from DotGitDirectory: C:\My\Path\To\Repository\.git - C:\My\Path\To\Repository
ERROR [09/27/16 18:29:37:64] An unexpected error occurred:
YamlDotNet.Core.YamlException: (Line: 1, Col: 1, Idx: 0) - (Line: 1, Col: 1, Idx: 0): Exception during deserialization ---> System.Runtime.Serialization.SerializationException: Property 'assembly-informational-version' not found on type 'GitVersion.Config'.
   em YamlDotNet.Serialization.TypeInspectors.TypeInspectorSkeleton.GetProperty(Type type, Object container, String name, Boolean ignoreUnmatched)
   em YamlDotNet.Serialization.NodeDeserializers.ObjectNodeDeserializer.YamlDotNet.Serialization.INodeDeserializer.Deserialize(EventReader reader, Type expectedType, Func`3 nestedObjectDeserializer, Object& value)
   em YamlDotNet.Serialization.ValueDeserializers.NodeValueDeserializer.DeserializeValue(EventReader reader, Type expectedType, SerializerState state, IValueDeserializer nestedObjectDeserializer)
   --- Fim do rastreamento de pilha de exceções internas ---
   em YamlDotNet.Serialization.ValueDeserializers.NodeValueDeserializer.DeserializeValue(EventReader reader, Type expectedType, SerializerState state, IValueDeserializer nestedObjectDeserializer)
   em YamlDotNet.Serialization.ValueDeserializers.AliasValueDeserializer.DeserializeValue(EventReader reader, Type expectedType, SerializerState state, IValueDeserializer nestedObjectDeserializer)
   em YamlDotNet.Serialization.Deserializer.Deserialize(EventReader reader, Type type)
   em YamlDotNet.Serialization.Deserializer.Deserialize[T](TextReader input)
   em GitVersion.ConfigSerialiser.Read(TextReader reader)
   em GitVersion.ConfigurationProvider.Provide(String workingDirectory, IFileSystem fileSystem, Boolean applyDefaults, Config overrideConfig)
   em GitVersion.ExecuteCore.ExecuteInternal(String targetBranch, String commitId, GitPreparer gitPreparer, IBuildServer buildServer, Config overrideConfig)
   em GitVersion.ExecuteCore.ExecuteGitVersion(String targetUrl, String dynamicRepositoryLocation, Authentication authentication, String targetBranch, Boolean noFetch, String workingDirectory, String commitId, Config overrideConfig)
   em GitVersion.SpecifiedArgumentRunner.Run(Arguments arguments, IFileSystem fileSystem)
   em GitVersion.Program.VerifyArgumentsAndRun()
INFO [09/27/16 18:29:37:64]
INFO [09/27/16 18:29:37:64] Attempting to show the current git graph (please include in issue):
INFO [09/27/16 18:29:37:64] Showing max of 100 commits
INFO [09/27/16 18:29:37:84] * 55d9a41 13 minutes ago  (HEAD -> develop, origin/develop)
* 8541f5c 15 minutes ago
* fc70d4e 19 minutes ago
* 6ea8134 31 minutes ago
* a8c4dca 64 minutes ago
* 50084c7 68 minutes ago
* 539f8b2 72 minutes ago
* a3b256a 83 minutes ago
* 1eaaa17 5 hours ago
* a2bcf0c 2 days ago
* 8ae274e 2 days ago
* 32134b0 2 days ago
* 80b728d 2 days ago
* 10adcc1 2 days ago
* db9637e 2 days ago
* 6919df6 2 days ago
* 9cecb89 2 days ago
* abc4b00 2 days ago
* 038792f 2 days ago
| *   844a110 2 days ago  (refs/stash)
| |\
|/ /
| * ce0779c 2 days ago
|/
* f315dbf 2 days ago  (origin/master, master)
```

**Comment by JakeGinnivan** [comment]
According to https://gitversion.readthedocs.io/en/latest/configuration/ it is `assembly-informational-format` not `assembly-informational-version`?

**Comment by brunobertechini** [comment]
I have tried that with no luck

Same problem

Bruno

**Comment by asbjornu** [comment]
@brunobertechini, when you change the property to `assembly-informational-format`, what does the error look like?

**Comment by mzaatar** [comment]
Hello @brunobertechini , I had the same problem as you but with different section "BranchConfig". After hours of investigation I found that there the build server was picking up an old version of GitVersion because when I changed the VSTS build agent from local to be hosted. It worked fine for me. I don't know how it picks a different version but will have deep investigation later. 
How I discovered that issue? I run the same build script on the local machine and working fine. And then form the local build server and works fine as well. But when I try to pick the get version config by running
```
 .\packages\GitVersion.CommandLine.3.6.4\tools\GitVersion /showConfig
```
it gave me parsing exception.
